ssr/src/fallback.rs

```diff
@@ -26,7 +26,7 @@ pub async fn file_and_error_handler(
         handler(req).await.into_response()
     }
 }
-
+// #[allow(unreachable_patterns)]
```

Comment:
lets remove this instead of commenting?

Reply:
sure

Reply:
done @rupansh-gob

Reply:
opened a new pr for this , to resolve build failure.
should i close this one?

Reply:
New PR faces build failure as well—do we need to run or add a Fly script manually?